pyproject.toml

```
@@ -1,9 +1,9 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools",
     "wheel",
-    "ninja; sys_platform != 'win32'",
-    "cmake>=3.14",
-    "pybind11~=2.6.1",
+    "scikit-build>=0.13",
```

```suggestion
    "scikit-build @ git+https://github.com/henryiii/scikit-build@henryiii/fix/logger",
```

(I've been known to get the syntax wrong, but I think that's it)